src/formatDate.js

```diff
@@ -7,8 +7,35 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const formatedDate = [];
+  const splitedDate = date.split(fromFormat[3]);
```

Comment:
The variable name 'splitedDate' contains a typo. It should be 'splitDate' to correctly represent the past tense of 'split'.